### PR TITLE
Added localization for title and category for dacpac

### PR DESCRIPTION
**Pull request by smartguest**
Added localization for the title and category in the package.json of dacpac.

This is to start the fix for: #6326

### CODE REVIEW

File: extensions/dacpac/package.nls.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
 	"dacFx.settings": "Dacpac",
+	"dacFx.startTitle": "Data-tier Application Wizard",
+	"dacFx.category": "Data-tier Application",
```
Comment (kisantia) [inline extensions/dacpac/package.nls.json:4]: it's a little long having the command show up as "Data-tier Application: Data-tier Application Wizard." Should we just get rid of the category or shorten it to DacFx or Dacpac? @dzsquared for other ideas

Comment (Charles-Gagnon) [reply extensions/dacpac/package.nls.json:4]: What about having the category be `DacFx`? That way it actually matches the extension name as well which makes more sense to me

Comment (dzsquared) [reply extensions/dacpac/package.nls.json:4]: I would support the category being set to either "DacFx" or "SqlPackage"
unfortunately the extension name is "SQL Server Dacpac" -> this is fairly misleading on both the supported platforms and functionality.

File: resources/xlf/en/dacpac.xlf [hunk 2]
```diff
@@ -4,6 +4,12 @@
     <trans-unit id="dacFx.settings">
       <source xml:lang="en">Dacpac</source>
     </trans-unit>
+    <trans-unit id="dacFx.startTitle">
+      <source xml:lang="en">Data-tier Application Wizard</source>
+    </trans-unit>
+    <trans-unit id="dacFx.category">
+      <source xml:lang="en">Data-tier Application</source>
```
Comment (kisantia) [inline resources/xlf/en/dacpac.xlf:11]: doesn't this also need to be updated to DacFx?

**Review by Charles-Gagnon — APPROVED** [review]

**Review by kisantia — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**